job-dsls/jobs/FlowJob.groovy

```diff
@@ -739,7 +739,7 @@ job("kie-docker-ci-images-${kieMainBranch}") {
     description("Builds CI Docker images for master branch. <br> IMPORTANT: Created automatically by Jenkins job DSL plugin. Do not edit manually! The changes will get lost next time the job is generated. ")
 
     parameters {
-        stringParam("kieVersion", "8.0.0-SNAPSHOT", "Please edit the version of the kie release <br> i.e. typically <b> major.minor.micro.EXT </b>i.e. 8.0.0.Beta1<br> Normally the kie version will be supplied by parent job <br> ******************************************************** <br> ")
+        stringParam("kieVersion", "7.7.0-SNAPSHOT", "Please edit the version of the kie release <br> i.e. typically <b> major.minor.micro.EXT </b>i.e. 7.7.0.Beta1<br> Normally the kie version will be supplied by parent job <br> ******************************************************** <br> ")
```

Comment:
Is 7.7.0-SNAPSHOT fine for 7.5.x branch?

Reply:
actually it is better than 8.0.0-SNAPSHOT, which doesn'ŧ exist or only by mistake I think

Reply:
nop - but this is only a text in the description - this value comes via sh